README.md

```diff
@@ -45,15 +45,54 @@ absolute_path = resolve_robotics_uri_py.resolve_robotics_uri("package://moveit_r
 
 `resolve_robotics_uri_py` also install a command line tool called `resolve-robotics-uri-py` for use in scripts, that can be used as:
 
-~~~
+~~~bash
 resolve-robotics-uri-py package://iCub/robots/iCubGazeboV2_7/model.urdf
 ~~~
 
 For example,  on bash this can be used to easily convert the a urdf specified via `package://` to an sdf (assuming you have Gazebo installed), using the [backtick operator](https://www.redhat.com/sysadmin/backtick-operator-vs-parens):
-~~~
+~~~bash
 gz sdf -p `resolve-robotics-uri-py package://iCub/robots/iCubGazeboV2_7/model.urdf`
 ~~~
 
+### Adding Custom Search Paths
+
+Some packages may not be installed in standard locations, or you may want to use a custom directory structure. In such cases, you have a few options:
+
+#### Option 1: Modify the Native Search Paths
+
+You can add the search path to one of the natively supported paths of Gazebo or ROS, such as:
+
+* `GAZEBO_MODEL_PATH`
+* `ROS_PACKAGE_PATH`
```

Comment:
```suggestion
* `GZ_SIM_RESOURCE_PATH`
* `AMENT_PREFIX_PATH`
```

this are used in Gazebo Classic and ROS 1 that are both deprecated, let's document the modern alternative.